cg/server/api.py

```diff
@@ -323,27 +323,27 @@ def analyses():
 @BLUEPRINT.route("/options")
 def options():
     """Fetch various options."""
-    customer_objs: Optional[models.Customer] = (
+    customers: Optional[Customer] = (
         db.Customer.query.all() if g.current_user.is_admin else g.current_user.customers
     )
 
-    apptag_groups = {"ext": []}
-    for application_obj in db.applications(archived=False):
-        if not application_obj.versions:
-            LOG.debug("Skipping application %s that doesn't have a price", application)
+    apptag_groups: Dict[str, list[str]] = {"ext": []}
+    for application in db.applications(archived=False):
```

Comment:
Does application require a type hint here?

Reply:
Python does not support type hints in for loop initialization. Then one have  to add it before the loop. We can fix this when doing the application refactoring